E2E-tests/secrets/substrate/local/local.json

```diff
@@ -0,0 +1,48 @@
+{
+	"db": {
+		"type": "postgresql",
+		"username": "postgres",
+		"password": "postgres",
+		"host": "127.0.0.1",
+		"port": "5432",
+		"name": "qa_demo",
+		"url": "${db.type}://${db.username}:${db.password}@${db.host}:${db.port}/${db.name}"
+	},
+	"dbSync": {
+		"type": "postgresql",
+		"username": "postgres",
+		"password": "postgres",
+		"host": "127.0.0.1",
+		"port": "5432",
+		"name": "cexplorer",
+		"url": "${dbSync.type}://${dbSync.username}:${dbSync.password}@${dbSync.host}:${dbSync.port}/${dbSync.name}"
+	},
+	"wallets": {
+		"faucet-0": {
+			"scheme": "ECDSA",
+			"address": "5C7C2Z5sWbytvHpuLTvzKunnnRwQxft1jiqrLD5rhucQ5S9X",
+			"secret_seed": "cb6df9de1efca7a3998a8ead4e02159d5fa99c3e0d4fd6432667390bb4726854",
+			"public_key": "020a1091341fe5664bfa1782d5e04779689068c916b04cb365ec3153755684d9a1"
+		},
+		"active-flow": {
```

Comment:
These 3 keys can be removed, we don't have active flow, nor passive and nor negative.